src/utilities/flattenRoutes.ts

```diff
@@ -1,31 +1,35 @@
 import { RouteFlat, Routes, isParentRoute, isNamedRoute } from '@/types'
+import { Path, combineParams, path as createPath } from '@/utilities'
 
-export function flattenRoutes(routes: Routes, path = ''): RouteFlat[] {
+export function flattenRoutes(routes: Routes, parentPath = '', parentParams = {}): RouteFlat[] {
```

Comment:
Yeah this signature looks right to me because it matches pretty closely how the types are generated.